resources/aws/security_group.go

```diff
@@ -181,3 +256,29 @@ func (s SecurityGroup) GetID() (string, error) {
 
 	return *securityGroup.GroupId, nil
 }
+
+func (s SecurityGroup) getGroupFilters() ([]*ec2.Filter, error) {
```

Comment:
It's because the filters are used in both the `findExisting` and `findGroupWithRule` methods. I'm going to go with this.